api/v1/pinning.go

```diff
@@ -234,6 +234,112 @@ func filterForStatusQuery(q *gorm.DB, statuses map[types.PinningStatus]bool) (*g
 	return q.Where("active or pinning or failed"), nil
 }
 
+// handleAddBatchedPins  godoc
+// @Summary      Add and pin objects in batches
+// @Description  This endpoint adds a pin to the IPFS daemon.
+// @Tags         pinning
+// @Accept		 json
+// @Produce      json
+// @Success      202	{object}  []types.IpfsPinStatusResponse
+// @Failure      500    {object}  util.HttpError
+// @in           202,default  string  Token "token"
+// @Param        pin          body      []types.IpfsPin  true   "Pin Body {[cid:cid, name:name]}"
+// @Param        ignore-dupes  query     string                   false  "Ignore Dupes"
+// @Param        overwrite	   query     string                   false  "Overwrite conflicting files in collections"
+// @Router       /pinning/batched-pins [post]
```

Comment:
nit: I personally don't like dashes on endpoint names. Maybe on v2 we could only have `POST /pinning/pins` and not two endpoints that essentially do the same thing.

Reply:
it won't be directly compatible with the original `pinning/pins` apiV1. hyphens (dash) on names are the common practice. IMO, its ok to stick with `/pinning/batched-pins`

Reply:
> Maybe on v2 we could only have POST /pinning/pins and not two endpoints that essentially do the same thing.

We can't do this, as it will break the pinning spec compliance

Reply:
> I personally don't like dashes on endpoint names

hyphens on URL is very common